etc/reset.py

```diff
@@ -172,9 +176,37 @@ def get_pachyderm(deploy_version):
     run("docker", "pull", "pachyderm/pachd:{}".format(deploy_version))
     run("docker", "pull", "pachyderm/worker:{}".format(deploy_version))
 
+def rewrite_config():
+    log.info("Rewriting config")
+
+    keys = set([])
+
+    try:
+        with open(os.path.expanduser("~/.pachyderm/config.json"), "r") as f:
+            j = json.load(f)
+    except:
+        return
+        
+    v2 = j.get("v2")
+    if not v2:
+        return
+
+    contexts = v2["contexts"]
+
+    for k, v in contexts.items():
+        if LOCAL_CONFIG_PATTERN.fullmatch(k) and len(v) > 0:
```

Comment:
This is for my info: Why only the local context? Why is this not a problem with a remote context?

Reply:
That would remove the ability to access remote clusters, since their contexts would be gone

Reply:
Isn't the intent here to reset everything -- be it remote or local? Or do we support reset only for local deployments which represents a dev/test kind of workflow?

Reply:
It's only for resetting local dev deployments